pkg/metrics_store/metrics_writer.go

```diff
@@ -46,7 +40,7 @@ func NewMultiStoreMetricsWriter(stores []*MetricsStore) MetricsWriter {
 //
 // WriteAll writes metrics so that the ones with the same name
 // are grouped together when written out.
-func (m MultiStoreMetricsWriter) WriteAll(w io.Writer) {
+func (m MetricsWriter) WriteAll(w io.Writer) {
```

Comment:
unrelated to this change, can we improve error handling in this function if the write fails?

Reply:
That a good idea, I added the error handling in a new commit